src/commands/task/run.ts

```diff
@@ -43,7 +43,7 @@ export default class Run extends Command {
     const fromCwd = (p: string) => path.join(process.cwd(), p);
 
     runScript(
-      fromCwd(`tasks/${args.task}.js`),
+      fromCwd(`tasks/${args.task}.ts`),
```

Comment:
Would it be possible to support both TypeScript or JavaScript? Don’t want to break existing projects.

Reply:
Possible approaches: 

- Framework generator, where you can create your initial setup with JS or TS
- Accept both TS and JS with a deprecation flag for JS (delete js support in x few versions)
- Accept both TS and JS with a fallback on JS (by default create the files .ts)

Which one you like? Or how would you like to approach accepting both js and ts?

Reply:
I like the last option, with the task name provided we can check to see which file extension exists in the repo.